README.md

```diff
@@ -184,6 +215,8 @@ Command Options
 | `--python=$path`                  | Set path to the python (2) binary
 | `--msvs_version=$version`         | Set Visual Studio version (win)
 | `--solution=$solution`            | Set Visual Studio Solution version (win)
+| `--gypjs`                         | Use gyp.js instead of gyp
```

Comment:
Perhaps mention that this can go away at any stage until we can take a decision about whether or not we should keep it. Both in readme and some smaller note here.